src/test/java/com/google/api/codegen/transformer/testdata/myproto.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
```

Comment:
Instead of adding new protos like this -- can we reuse existing library.proto?
It has lots of complicated enum values for testing.